src/main/resources/i18n.properties

```diff
@@ -0,0 +1,407 @@
+uno.draw.played=You drew and played a {0}
+
+uno.play.description=Play a card from your hand, pick the color for a wildcard immediately
+
+uno.play.error=You need to play a valid card that's in your hand
+
+uno.win={0} played a **{1}** and won **{2}** credits by winning the game
+uno.win.short={0} won the game and won **{1}** credits
+uno.win.you=You played a **{0}** and won, you won **{1}** credits
+uno.lost=You lost **{0}** credits
+uno.turn=It's your turn!
+uno.draw=You had to draw {0} cards because {1} played a {2}
+uno.end=The game of uno has concluded
+
+uno.help=Run {0}help to show which commands you can use
+# Adminabuse
+give.description=Give coins to someone
+
+give.everyone=Everyone received **{0}** credits
+give.success={0} now has **{1}** credits
+
+# Daily
+daily.description=Collect your daily credits
+daily.wait=You need to wait {0, choice,1#hours|hours} and {1,choice, 1#minute|minutes} before you can collect your next credits
+daily.success=You collected your daily **{0} credits** \
+Your new balance is now **{1} credits**
+
+daily.event.collecting=You're collecting your daily christmas gift
+daily.event.collected=You collected your daily christmas gift
+daily.event.falling=Your gift is falling down the chimney, it contains:\
+{0}{1}{2}{3}{4}{5}{6} **credits**
+daily.event.fell=Your gift fell down the chimney, it contained:\
+{0}{1}{2}{3}{4}{5}{6} **credits**
+daily.event.balance=Your new balance is now **{0} credits **
+
+# Weekly
+weekly.description=Collect your weekly credits
+
+weekly.wait=You need to wait {0, choice, 1# day| days},  {1, choice,1# hours| hours} and {2,choice, 1# minute| minutes} before you can collect your next credits
+weekly.success=You collected your weekly **{0} credits** \
+Your new balance is now **{1} credits**
+
+# Records
+records.description=Show all records, records for one member or one record. List all possible records using the `list` argument
+
+
+records.list=Records list
+records.person={0}'s Records
+records.no_records.person=No records yet for {0}
+records.no_records=There are no records yet, claim credits and play a game to start the records
+records.global=Global Casino Records
+records.local=Casino Records
+
+
+records.error.valid={0} is not a valid member name or record name
+records.error.args=This commands takes max 2 optional arguments. \
+{0}\
+If the name of the member consists of multiple words, put it between quotes for it to be recognised as a name.
+
+# Balance
+balance.description=Show your current credit balance
+
+balance=Your current balance is **{0}**
+
+# Roleassign
+
+roleassign.name=roleassign
+roleassign.description=Display Role picker
+
+roleassign.embed.title={0} Roles
+roleassign.embed.description=Get your {0} roles here, react to get the role
+
+roleassign.add.description=Add a role to the role assigner
+
+roleassign.error.no_category=No category given
+roleassign.error.no_emoji=No emoji, role and name given
+roleassign.error.no_role=No role and name given
```

Comment:
Hier opnieuw "No role was given, but name was" of "No role or name was given" afhankelijk van betekenis